index.html

```diff
@@ -176,30 +176,30 @@
                 <li class="header">{{ 'MAIN_NAV' | translate }}</li>
                 <!-- Home Page -->
                 <li ng-class="getClass('/')">
-                    <a href="#/">
+                    <a ng-href="/admin">
```

Comment:
This could also be `./`, to better support the edge case where the user installs the interface in a different folder than `admin`.